### DISCUSSION TITLE
lolNvidiaOC

### DISCUSSION THREAD

**Discussion by Majd00shka** [Lolliedieb/lolMiner-releases#1624] [Q&A]
Good day everyone,

I've noticed the beta release of lolNvidiaOC.exe and I'm having a problem setting it up. I tried to edit the bat file I use for dual mining but was always unsucsesful.

here is my codes that I'm using in my bat file:

@echo off

setlocal enableDelayedExpansion

Rem #################################
Rem ## Begin of user-editable part ##
Rem #################################

set "POOL="
set "WALLET="

set "TONPOOL="
set "TONWALLET="
							
set "EXTRAPARAMETERS="

Rem #################################
Rem ##  End of user-editable part  ##
Rem #################################

if exist "%CD%\lolMiner.exe" goto infolder
echo "Searching for lolMiner.exe, because is not in this folder.That could take sometime..."
for /f "delims=" %%F in ('dir /b /s "C:\lolMiner.exe" 2^>nul') do set MyVariable=%%F
if exist "%MyVariable%" goto WindowsVer
echo "lolMiner.exe is not found in the system, that could be blocked by Windows Defender or Antivirus "
goto END

:infolder
set MyVariable=%CD%\lolMiner.exe

:WindowsVer
echo "Running lolMiner from %MyVariable%"
for /f "tokens=4-5 delims=. " %%i in ('ver') do set VERSION=%%i.%%j
if "%version%" == "10.0" goto W10
goto OtherW

:W10
"%MyVariable%" --algo ETHASH --pool !POOL! --user !WALLET! --dualmode TONDUAL --dualpool !TONPOOL! --dualuser !TONWALLET! !EXTRAPARAMETERS! --watchdog exit
timeout 10
goto W10
goto END

:OtherW
"%MyVariable%" --algo ETHASH --pool !POOL! --user !WALLET! --dualmode TONDUAL --dualpool !TONPOOL! --dualuser !TONWALLET! !EXTRAPARAMETERS! --watchdog exit --nocolor
timeout 10
goto OtherW

:END
pause

I already but the file lolNvidiaOC.exe in the same folder. Can anyone help me out on how to set up the following?

lolNvidiaOC.exe --cclk 1605 --delay 20 --omclk 4325 --fan 90

**Comment by rich2551** [comment]
Put your code "lolNvidiaOC.exe --cclk 1605 --delay 20 --omclk 4325 --fan 90" under the @echo off command and that should do it.
